### ISSUE TITLE
Map disposition recommendation strings to CRM enums

### ISSUE THREAD

**Issue by godfreyyeung** [NYCPlanning/labs-zap-search#903]
Map disposition recommendation strings to CRM enums:

```
Options:
717170000: Approved
717170001: Approved with Modifications/Conditions
717170002: Disapproved
717170003: Disapproved with Modifications/Conditions
717170004: Non-Complying
717170005: Vote Quorum Not Present
717170006: Received after Clock Expired
717170007: No Objection
717170008: Waiver of Recommendation
Default: N/A
```

https://dcppfsuat2.crm9.dynamics.com/api/data/v9.0/EntityDefinitions(LogicalName='dcp_communityboarddisposition')/Attributes

**Comment by godfreyyeung** [comment]
Addressed by #902

**Comment by andycochran** [comment]
Can this be closed?
